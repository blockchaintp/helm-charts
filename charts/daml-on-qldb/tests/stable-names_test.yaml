suite: daml-on-qldb.test for stable names
templates:
  - service.yaml
  - jsonapi-service.yaml
  - trigger-service.yaml
  - statefulset.yaml
  - serviceaccount.yaml
  - cert-job.yaml
release:
  name: test
  namespace: testns
tests:
  - it: check for stable entity names (services, statefulsets, serviceaccounts)
    asserts:
      - matchSnapshot:
          path: metadata.name
