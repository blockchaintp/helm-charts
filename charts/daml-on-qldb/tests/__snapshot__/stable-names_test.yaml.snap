check for stable entity names (services, statefulsets, serviceaccounts):
  1: |
    test-daml-on-qldb
  2: |
    test-daml-on-qldb-jsonapi
  3: |
    test-daml-on-qldb-triggers
  4: |
    test-daml-on-qldb
  5: |
    test-daml-on-qldb
  6: |
    test-daml-on-qldb-cert
